### ISSUE TITLE
Updated Prior

### ISSUE THREAD

**Issue by saucetray** [stripe/rainier#506]
Is there a way in this library to use the model you ran inference on as your new prior?

**Comment by saucetray** [comment]
I saw someone thumbs this up but I don't see anything in the documentation explaining using your current model as a prior.

**Comment by avibryant** [comment]
No, it's not possible. We don't have a closed-form representation of the posterior, which we need to use something as a prior.
